scripts/setup-mongo-dev.sh

```diff
@@ -1,15 +1,17 @@
 #! /bin/bash
```

Comment:
Would it be OK to remove this script for native development in favor of `scripts/ingest-local-setup.sh` for Dockerized development?  The latter basically adapts the former to use a name and approach that's more consistent with those in Core (i.e. [rails_local_setup.rb](https://github.com/broadinstitute/single_cell_portal_core/blob/217c21ed41e52b4489a5d097cd816091d311745a/rails_local_setup.rb)), and is executed by default in the Docker setup.

If this file can be removed, I would also update references to it in `ingest_pipeline.py` and `README.md`.

Reply:
Yes, it looks like `ingest-local-setup.sh` has all the elements of `setup-mongo-dev.sh` so the latter is no longer needed. My only [issue](https://github.com/broadinstitute/scp-ingest-pipeline/pull/290/files#r1085420240) is around any security concerns regarding `config/.ingest_env`